lib/high_voltage/configuration.rb

```diff
@@ -39,6 +39,14 @@ def page_caching=(value)
       @page_caching = value
     end
 
+    def page_ids
+      HighVoltage::PageCollector.new(HighVoltage.full_path).page_ids
+    end
+
+    def full_path
+      Rails.root.join('app', 'views', HighVoltage.content_path)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.